### PR TITLE
Create dependabot.yml

### PR DESCRIPTION
**Pull request by niklaslong**
This will check for dependency updates daily (only for Mix). 

https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "mix" # See documentation for possible values
```
Comment (oliverbarnes) [inline .github/dependabot.yml:8]: should we add `npm` as well for updates to js packages, like the last one for lodash?

Comment (niklaslong) [reply .github/dependabot.yml:8]: I was unsure, we could but that might get _very_ noisy... 🤔

Comment (niklaslong) [reply .github/dependabot.yml:8]: We may also be able to rely on [Phoenix keeping those sufficiently up to date for us](https://github.com/phoenixframework/phoenix/pull/3932).

Comment (oliverbarnes) [reply .github/dependabot.yml:8]: works for me and we can revisit later

**Comment by niklaslong** [comment]
Btw, last time run will be displayed [here](https://github.com/liquidvotingio/auth/network/updates).

**Pull request merged**